packages/local/src/fork-foreman.ts

```diff
@@ -12,3 +13,21 @@ export function fork(argv: string[]): Promise<void> {
     })
   })
 }
+
+// depending if this is being ran before or after compilation
+// we need to check for `.ts` and `.js` extensions and use
+// the appropriate one.
+function getForemanScriptPath() {
+  let file = 'run-foreman'
+
+  let withJsExtension = path.join(__dirname, file + '.js')
```

Comment:
@chadian it looks like most of the `let` can be changed to `const`

Reply:
👍